### ISSUE TITLE
npm audit

### ISSUE THREAD

**Issue by marioedgar** [danielkellyio/awake-template#16]
Im see security vulnerabilities on npm install:

<img width="685" alt="Screen Shot 2020-01-02 at 8 34 13 PM" src="https://user-images.githubusercontent.com/1877708/71707553-5f4aa980-2d9f-11ea-9c15-ff93b8baa17d.png">

**Comment by danielkellyio** [comment]
See if an ‘npm audit —fix’ takes care of it. If so feel free to submit a PR. Thanks!

**Comment by marioedgar** [comment]
OK so i took a stab at this. Looks like the audit fix will cause a mismatch in vue.js versions:

Steps i took:

```
npm audit fix
...
...
fixed 20 of 20 vulnerabilities in 20676 scanned packages
```

Ran the project:

```
npm run dev
```

I now get the following error:

![Screenshot from 2020-01-23 19-17-50](https://user-images.githubusercontent.com/1877708/73042186-27220e00-3e15-11ea-8f5e-3ee63525a3f0.png)

The first thing i tried after this, was to update nuxt.js

```
npm i nuxt@latest --save
```

I got the same mismatch error after this as well.

**Comment by marioedgar** [comment]
ok, I got this fixed. I tried to open a pull request but it said i dont have permission. Here are the steps i did to fix:

```
npm audit fix
npm install vue@latest --save
npm install vue-server-renderer@latest --save
```

According to this, just need to lock down the versions of vue and the server renderer:

https://github.com/facebook/jest/issues/3254

Side note - Running this on linux gave me some errors due to too many watchers. Not sure if thats my system or the code.
